src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: $calendar-width;
```

Page behaves strange in the narrow viewport, try it in developer tools. Probably would be better to set `width` instead of `max-width`
```suggestion
  width: $calendar-width;
```

---

It's the same situation with width

---

Are you sure? Because, when I tried it in the dev tools it's behavior was different